specs/models/relations/artifact_wrapper_spec.py

```diff
@@ -62,16 +63,16 @@
                 self.artifact = ArtifactBuilder()
                 self.artifact.build_with(file_url='asdf', type='image')
                 self.artifact.save()
-                self.artifact.update_with(tags=['a', 's', 'd', 'f'])
+                self.artifact.update_with(user_tags=['a', 's', 'd', 'f'])
```

Comment:
Do you mind adding some tests for the other tag types as well here?

Reply:
The other tag types are actually tested in the image_recognition_spec.py which is where they are used as well.

Reply:
done though.